types/react-i18next/src/context.d.ts

```diff
@@ -1,12 +1,18 @@
-import { i18n } from "i18next";
+import { i18n, TranslationFunction, ReactOptions } from "i18next";
+import React = require('react');
 
 export interface ReactI18NextOptions {
     wait?: boolean;
     withRef?: boolean;
     bindI18n?: string;
     bindStore?: string;
+    /**
+     * @deprecated This option is now ignored and the function name is always `t`
+     */
     translateFuncName?: string;
```

Comment:
I could change its type to `'t'` or just remove it altogether. I only kept it because it still technically exists in the return from `getDefaults()`.